homeassistant/components/uptime/sensor.py

```diff
@@ -13,50 +13,43 @@
 
 DEFAULT_NAME = "Uptime"
 
-ICON = "mdi:clock"
-
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
-    {
-        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
-        vol.Optional(CONF_UNIT_OF_MEASUREMENT, default="days"): vol.All(
-            cv.string, vol.In(["minutes", "hours", "days", "seconds"])
-        ),
-    }
+PLATFORM_SCHEMA = vol.All(
+    cv.deprecated(CONF_UNIT_OF_MEASUREMENT, invalidation_version="0.119"),
```

Comment:
We don't need to have an invalidation version in this case, as it already is set to be a noop in this PR.